oauth2_provider/views.py

```diff
@@ -11,7 +11,8 @@
 import provider.oauth2.forms
 import provider.scope
 from provider.oauth2.models import AccessToken
-from provider.oauth2.views import OAuthError, Capture, Redirect  # pylint: disable=unused-import
+from provider.oauth2.views import OAuthError, OAuth2AccessTokenMixin
+from provider.oauth2.views import Capture, Redirect  # pylint: disable=unused-import
```

Comment:
the reason i separated the imports from `provider.oauth2.views` is because `Capture` and `Redirect` are imported from this module by (something...tests break without them) but not used locally.  So I only wanted the pylint directive to apply to those.